src/augmentation.py

```diff
@@ -1,5 +1,32 @@
+import cv2
+import numpy as np
+import imgaug as ia
 from imgaug import augmenters as iaa
 
+
+class PerspectiveTransform(iaa.PerspectiveTransform):
```

Comment:
@taraspiotr I am not sure if I like that we are inheriting from implementation class and the interface iaa.Augmenter class. There is a tiny bit of boiler-plating involved if we chose to do it by the book but I'd still go for it.

Reply:
@jakubczakon inheriting from Augmenter would actually mean quite a lot of copy pasting, what do you think about this kind of monkey patch:
`iaa.PerspectiveTransform._augment_images = _perspective_transform_augment_images`?

Reply:
@taraspiotr I don't have a strong opinion on monkey patching but I don't mind it here since this is exactly what we are doing -> patching buggish implementation